src/init/starship.tcsh

```diff
@@ -0,0 +1,14 @@
+setenv STARSHIP_SHELL tcsh;
+setenv STARSHIP_SESSION_KEY `(::STARSHIP:: session)`;
+alias precmd 'set STARSHIP_CMD_STATUS = $status;set NUM_JOBS=`(jobs | wc -l)`;set STARSHIP_END_TIME = `::STARSHIP:: time`;set STARSHIP_DURATION = 0;if ( $?STARSHIP_START_TIME ) @ STARSHIP_DURATION = $STARSHIP_END_TIME - $STARSHIP_START_TIME;set prompt = "`(::STARSHIP:: prompt --status $STARSHIP_CMD_STATUS --jobs $NUM_JOBS --cmd-duration $STARSHIP_DURATION)`";set prompt = "\n$prompt[1] \n$prompt[2]";unset STARSHIP_START_TIME;';
```

Comment:
From what I've seen disabling newlines is among the most popular things people change in their configs. Is a `for` loop not possible here?

Reply:
a for loop is unfortunately not possible there, but I realized all I had to do was escape the newlines from within starship. So there are no longer any limitations on newlines in my latest revision.